cli/lib/tasks/download.js

```diff
@@ -54,6 +54,20 @@ const getBaseUrl = () => {
   return defaultBaseUrl
 }
 
+const getCA = () => {
+  let ca
+
+  if (util.getEnv('CYPRESS_DOWNLOAD_MIRROR_CA')) {
```

Comment:
How does this read from `.npmrc`?

Reply:
When node runs it reads `.npmrc` and sets values on global variable - `process.env.npm_config_*`.
`util.getEnv` already leverages this to retrieve other env settings: https://github.com/cypress-io/cypress/blob/develop/cli/lib/util.js#L455